tests/journald_tests.cpp

```diff
@@ -312,7 +313,7 @@ TEST_F(JournaldLoggerTest, ROOT_LogToJournaldWithBigLabel)
   AWAIT_READY(secondQuery);
   ASSERT_FALSE(strings::contains(secondQuery.get(), specialString));
 }
-#endif // __WINDOWS__
+#endif // __linux__
```

Comment:
Please either add  a matching `#ifdef`, or remove this (depending on whether the issue with CMAKE_BUILD_TYPE exists/remains or not).

Reply:
the matching `#ifdef` starts at line 184; that used to be misprint that I fixed.